src/services/local/index.js

```diff
@@ -105,7 +103,7 @@ export class Service {
 }
 
 export default function(options){
-  options = Object.assign({}, defaults, options);
+  options = Object.assign({ passReqToCallback: true }, defaults, options);
```

Comment:
Ya this is pretty common when dealing with passport, especially if you need to do account consolidation. We're already doing this in the OAuth2 service. @saiichihashimoto let's move this up into the `defaults` object, that way all the defaults stay organized together.